pkg/eventsapi/event_container.go

```diff
@@ -0,0 +1,30 @@
+package eventsapi
+
+import (
+	"encoding/json"
+)
+
+type EventContainer struct {
+	EventVersion EventVersion
+	EventData    json.RawMessage
+}
+
+func (ec *EventContainer) UnmarshalEvent() (Event, error) {
+	jsonEventData, err := ec.EventData.MarshalJSON()
```

Comment:
Is this marshal actually getting us anything?  I think we can skip it since `json.RawMessage` is already JSON...

Reply:
You right! I had some tests failing because my json syntax was wrong, fixed here: 620b735